scripts/jenkins/cloud/ansible/deploy-caasp-v4-terraform.yml

```diff
@@ -0,0 +1,568 @@
+#
+# (c) Copyright 2020 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# This playbook supports deployment of caaspv4 using terraform
+# against SOC (ardana) deployments.
+# Once the playbook finishes successfully, you can login into the
+# management machine from the deployer.
+# On deployer node: ssh -i caaspmanagementkey.pem sles@<managementmachinefloatingip>
+# Once you are in the management machine, the caasp bits
+# are in /home/sles/caasp/deployment/openstack.
+---
+- name: Setup Management Machine to Deploy CaaSPv4 using terraform
+  hosts: "{{ cloud_env }}"
+  gather_facts: True
+  remote_user: ardana
+  vars:
+    ca_path: "/etc/pki/trust/anchors/ca-bundle.pem"
+    external_net: "ext-net"
+    soc_home: "/var/lib/ardana"
+    image_url: "http://ibs-mirror.prv.suse.net/install/SLE-15-SP1-JeOS-QU2/SLES15-SP1-JeOS.x86_64-15.1-OpenStack-Cloud-QU2.qcow2"
+    image_path: "{{ soc_home }}/SLES15-SP1-JeOS.x86_64-15.1-OpenStack-Cloud-QU2.qcow2"
+    image_name: "SLE-15-SP1-JeOS-QU2"
+    keyname: "caaspmanagementkey"
+    management_machine_size: "m1.small"
+    management_machine_name: "caasp_management"
+    management_machine_net: "caasp-mm-net"
+    management_machine_router: "caasp-mm-router"
+    management_machine_subnet: "caasp-mm-subnet"
+    management_machine_subnet_cidr: "2.2.2.0/24"
+    management_machine_gateway: "2.2.2.1"
+    management_machine_secgroup: "caasp-mm-sec-group"
+    octavia_enabled: true
+    os_ca_path: "/etc/ssl/ca-bundle.pem"
+    service_osrc: "{{ soc_home }}/service.osrc"
+    ssh_opts: '-o BatchMode=yes -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o LogLevel=error -o ConnectTimeout=30'
+    service_caasp_osrc: "service_caasp.osrc"
+  tasks:
+
+  #for SOC 8: Use the regular Neutron LBaaS API
+  #conditional check ansible_distribution_release == "3" means SLES12SP3 deployer
+  - set_fact:
```

Comment:
Do you really need that task? What about setting this as variable, like:
`
octavia_enabled: "{{ not (ansible_distribution_release == '3') }}"
`